dotnet/src/Experimental/Process.Abstractions/KernelProcessUtilities.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Microsoft.SemanticKernel.Process;
+
+/// <summary>
+/// Shared Utilities used by Kernel Process Components
+/// </summary>
+public static class KernelProcessUtilities
```

Comment:
Can this be merged with the `InternalUtilities/process` that Chris setup yesterday?

Reply:
This is a good idea.  A couple thoughts:

1. - Including `RuntimeUtilities.props` into the `Abstractions` or `Core` project will cause a build error in the unit-tests, which have internal resolution to all projects.

This could be solved by having project specific unit-test project, but i'm thinking that,

2. `Abstractions` likely doesn't want to import all the internals that are targeted for the _runtimes_.  Introducing a second build properties file that only imports what is targeted for abstractions might be appropriate: `AbstractionUtilities.props`...with even potentially introducing sub-folders under `dotnet\src\InternalUtilities\process`.

-or-

3. If `KernelProcessUtilities` aren't required outside of `Absractions`, couldn't they simply be declared internal?

4. If we do need to share code between the runtimes and abstractions, what I setup may require some additional consideration (due to the type conflicts in the unit-tests).

Reply:
Thanks to @crickman for the shared resources refactor, after the merge incorporated the changes in the PR